test/functional/apps/discover/_collapse_expand.js

```diff
@@ -26,64 +26,51 @@ export default function ({ getService, getPageObjects }) {
   const PageObjects = getPageObjects(['common', 'discover', 'header']);
 
   describe('discover tab', function describeIndexTests() {
-    before(function () {
+    before(async function () {
       const fromTime = '2015-09-19 06:31:44.000';
       const toTime = '2015-09-23 18:31:44.000';
 
       // delete .kibana index and update configDoc
-      return kibanaServer.uiSettings.replace({
+      await kibanaServer.uiSettings.replace({
         'dateFormat:tz': 'UTC',
-        'defaultIndex': 'logstash-*'
-      })
-        .then(function loadkibanaIndexPattern() {
-          log.debug('load kibana index with default index pattern');
-          return esArchiver.load('discover');
-        })
+        defaultIndex: 'logstash-*',
+      });
+
+      log.debug('load kibana index with default index pattern');
+      await esArchiver.load('discover');
+
       // and load a set of makelogs data
-        .then(function loadIfEmptyMakelogs() {
-          return esArchiver.loadIfNeeded('logstash_functional');
-        })
-        .then(function () {
-          log.debug('discover');
-          return PageObjects.common.navigateToApp('discover');
-        })
-        .then(function () {
-          log.debug('setAbsoluteRange');
-          return PageObjects.header.setAbsoluteRange(fromTime, toTime);
-        });
+      await esArchiver.loadIfNeeded('logstash_functional');
+
+      log.debug('discover');
+      await PageObjects.common.navigateToApp('discover');
+
+      log.debug('setAbsoluteRange');
+      await PageObjects.header.setAbsoluteRange(fromTime, toTime);
     });
 
     describe('field data', function () {
-      it('should initially be expanded', function () {
```

Comment:
I know this wasn't part of your PR, but could you change this `describe` to something like `collapse expand`?  There's already a _field_data.js file that has `field data` in it's describe.